topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -116,17 +117,13 @@ In the second part of the tutorial, read counts of all 7 samples are used to ide
 >    > ```
 >    {: .comment}
 >
-> 3. Rename each dataset according to the sample id (e.g. `GSM461177_1`)
->
->    {% snippet faqs/galaxy/datasets_rename.md %}
->
-> 4. Check that the datatype is `fastqsanger` (e.g. **not** `fastq`). If it is not, please change the datatype to `fastqsanger`.
+> 3. Check that the datatype is `fastqsanger` (e.g. **not** `fastq`). If it is not, please change the datatype to `fastqsanger`.
 >
 >    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="fastqsanger" %}
 >
-> 5. Add to each dataset a tag corresponding to the name of the sample (`#GSM461177` or `#GSM461180`)
+> 4. Create a paired collection named `2 PE fastqs`, rename your pairs with the sample name followed by the attributes: `GSM461177_untreat_paired` and `GSM461180_treat_paired`.
```

Comment:
Does make sense to simplify the little bit sample names? I'm not sure if including "_paired" provides meaningful information at all.

Reply:
Ahh I see, you are also using single datasets; then it is completely fine.

Reply:
Perhaps using labels and simplifying the sample names could make the tutorial easier to follow.